Tools/TestWebKitAPI/Tests/WTF/CompactVariant.cpp

```diff
@@ -83,23 +83,39 @@ TEST(WTF_CompactVariant, Pointers)
 {
     int testInt = 1;
     float testFloat = 2.0f;
+    void* testPtr = reinterpret_cast<void*>(0x2600'1234'5678'9abc);
```

Comment:
What does this do on 32-bit platforms?

Reply:
Yeah this test is only applicable on 64-bit platforms. I separated it out to a test gated with `CPU(ADDRESS64)` so it's only built on 64-bit platforms.